website/sidebars.json

```diff
@@ -3,7 +3,7 @@
     "Introduction": ["intro"],
     "Validator": [
       "validator/setup",
-      "validator/troubleshooter"
+      "validator/troubleshooting"
```

Comment:
I used "troubleshooter" because I want parallelism of nouns in the outline, eg rewards, setup, troubleshooter, faq, etc.  Mixing nouns and gerunds prevents parallelism in the writing.

Reply:
Okay usually way I see in various docs it is mostly used as __troubleshooting__. It is intentional I'll revert it